plugins/providers/metabase/client.go

```diff
@@ -133,11 +135,22 @@ func (c *client) GetDatabases() ([]*Database, error) {
 	}
 
 	var databases []*Database
-	if _, err := c.do(req, &databases); err != nil {
-		return nil, err
-	}
+	var response interface{}
+	if _, err := c.do(req, &response); err == nil {
```

Comment:
let's keep handling error by `err != nil` to keep the code style consistency

Reply:
check now